vnext/Microsoft.ReactNative/Fabric/Composition/TextInput/WindowsTextInputComponentView.cpp

```diff
@@ -566,6 +567,194 @@ int64_t WindowsTextInputComponentView::sendMessage(uint32_t msg, uint64_t wParam
   return Super::sendMessage(msg, wParam, lParam);
 }
 
+WPARAM PointerPointToPointerWParam(const winrt::Microsoft::ReactNative::Composition::Input::PointerPoint &pp) noexcept {
+  WPARAM wParam = pp.PointerId();
+  wParam |= (POINTER_MESSAGE_FLAG_NEW << 16);
+  auto ppp = pp.Properties();
+  if (ppp.IsInRange()) {
+    wParam |= (POINTER_MESSAGE_FLAG_INRANGE << 16);
+  }
+  if (pp.IsInContact()) {
+    wParam |= (POINTER_MESSAGE_FLAG_INCONTACT << 16);
+  }
+  if (ppp.IsLeftButtonPressed()) {
+    wParam |= (POINTER_MESSAGE_FLAG_FIRSTBUTTON << 16);
+  }
+  if (ppp.IsRightButtonPressed()) {
+    wParam |= (POINTER_MESSAGE_FLAG_SECONDBUTTON << 16);
+  }
+  if (ppp.IsMiddleButtonPressed()) {
+    wParam |= (POINTER_MESSAGE_FLAG_THIRDBUTTON << 16);
+  }
+  if (ppp.IsXButton1Pressed()) {
+    wParam |= (POINTER_MESSAGE_FLAG_FOURTHBUTTON << 16);
+  }
+  if (ppp.IsXButton2Pressed()) {
+    wParam |= (POINTER_MESSAGE_FLAG_FIFTHBUTTON << 16);
+  }
+  if (ppp.IsPrimary()) {
+    wParam |= (POINTER_MESSAGE_FLAG_PRIMARY << 16);
+  }
+  if (ppp.TouchConfidence()) {
+    wParam |= (POINTER_MESSAGE_FLAG_CONFIDENCE << 16);
+  }
+  if (ppp.IsCanceled()) {
+    wParam |= (POINTER_MESSAGE_FLAG_CANCELED << 16);
+  }
+  return wParam;
+}
+
+WPARAM PointerRoutedEventArgsToMouseWParam(
```

Comment:
Do we want this helper somewhere where it can be used, rather than dedicated to WindowsTextInputComponentView?

Reply:
Hopefully we're not converting back to window messages regularly. -- If it something we need other places we can extract it then.